### PR TITLE
ST-299 Implement Check your answers

### PR DESCRIPTION
**Pull request by Shailesh-walke**
### Implemented Check your answers screen ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Review by paul-pearson — APPROVED** [review]
LGTM

**Review by osheaf01 — CHANGES_REQUESTED** [review]
One literal looks like a rogue commit?

File: src/main/steps/edge-case/check-your-answers/utils.test.ts [hunk 1]
```diff
@@ -359,37 +362,37 @@ describe('Form Summary > check-your-answers', () => {
...
-                    visuallyHiddenText: 'Are you named as the applicant on the application form you are submitting?',
+                    visuallyHiddenText: 'Subject’s name',
                   },
                 ],
               },
-              key: { text: 'Are you named as the applicant on the application form you are submitting?' },
-              value: { text: 'No' },
+              key: { text: 'Subject’s name' },
+              value: { text: 'Joe Bob' },
             },
```
[the remark this answers is not in the document]
Comment (Shailesh-walke) [reply src/main/steps/edge-case/check-your-answers/utils.test.ts:389]: updated it

**Comment by Shailesh-walke** [comment]
> One literal looks like a rogue commit?

Fixed

**Review by osheaf01 — APPROVED** [review]
LGTM
